plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -453,7 +453,7 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
   private fun convertAnnotationsToFeatures(annotations: Collection<T>): List<Feature> =
     annotations.map {
       it.setUsedDataDrivenProperties()
-      Feature.fromGeometry(it.geometry, it.getJsonObjectCopy())
+      Feature.fromGeometry(it.geometry, it.getJsonObjectCopy(), it.featureIdentifier)
```

Comment:
@Chaoba do we use generate ID for annotation plugin?

Reply:
We can update Annotation to let the id starts from a big number instead of 0

Reply:
that's still imho pretty unreliable, PTAL at my update here

Reply:
The new solution LGTM.